File: responses/8QRTD9-GGS/sum_of_certain_multiples.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(x):
    # write your code in Python
    N = []

    sum = 0

    for i in range(len(N)):
        if N[i] < x:
            num= N[i]
            if num % 5 == 0 and num != 5 and num != 3:
                sum += num
    return sum

File: responses/4T59SN-Z8Q/sum_of_certain_multiples.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(x) {
    // write your code in JavaScript
    var sum =0;
    for(var i=0; i<x; i++){
        if(i%15==0) continue;
        else if(i%3==0 || i%5==0){
            sum+=i;
        }
    }
    return sum;
}
